lib/ok_computer/rack_server.rb

```diff
@@ -1,8 +1,10 @@
 module OkComputer
   module RackServer
     def self.run!(port: 3000)
+      require "rack/handler/puma"
```

Comment:
I think I remember doing this on Somleng: https://github.com/somleng/somleng/blob/develop/lib/ok_computer/rack_server.rb

Reply:
Ah, good. I didn't notice it, but we just need to require it. Not sure why do we need to do rackup there?